pyccel/ast/numpyext.py

```diff
@@ -76,8 +80,8 @@
     'Ones',
     'OnesLike',
     'Product',
-    'PyccelArraySize',
     'Rand',
+    'RandInt',
```

Comment:
I think at some point we will want that every class in the `numpyext` module has a name starting with "Numpy". So it may be best to call this new class `NumpyRandint`.

Reply:
Sorry for being pedantic.. can we have it with no capital "I"?